### PR TITLE
Move containerd archives from code to yaml

### PR DESCRIPTION
**Pull request by afbjorklund**
This makes it more similar to update compared to the images, and allows for integrating with other archives in the future*.

* #1123

### CODE REVIEW

**Comment by afbjorklund** [comment]
For instance, for newer versions of podman:

https://github.com/afbjorklund/podman-full

**Comment by afbjorklund** [comment]
Possibly the version should be removed from the yaml, and be made a part of the filename instead ?

- containerd-1.7.6.yaml
- containerd-2.0.0-beta.5.yaml

Similar to the change made for the "image.yaml".

File: pkg/limayaml/defaults.go [hunk 1]
```diff
@@ -68,25 +70,21 @@ func defaultCPUType() CPUType {
 	return cpuType
 }
 
+//go:embed containerd.yaml
+var defaultContainerdYAML []byte
+
+type ContainerdYAML struct {
+	Version  string
+	Archives []File
+}
```
[the remark this answers is not in the document]
Comment (afbjorklund) [reply pkg/limayaml/defaults.go:79]: Good point, it should probably not be exported from defaults.go - if anything, exported from limayaml.go

[the remark this answers is not in the document]
Comment (afbjorklund) [reply pkg/limayaml/defaults.go:79]: Exporting it can wait until "step two", where it is possible to change archives using a more simple config

**Review by alexandear — APPROVED** [review]

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Review by AkihiroSuda — APPROVED** [review]

**Comment by AkihiroSuda** [comment]
Sorry, needs rebase again

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**